### PR TITLE
[Doc] Add Chocolatey info for Windows

### PR DESCRIPTION
**Pull request by mom1**
How to install in Windows using Chocolatey

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -83,6 +84,10 @@ This installer has additional installation wizard and includes static OpenSSL li
  1. Directly download the installer.
  2. Run the installer and follow the steps to install it on your system.
 
+#### <a name="chocolatey"></a>Using [Chocolatey](https://chocolatey.org)?
```
Comment (PeterDaveHello) [inline README.md]: I suppose this should be `Using [Chocolatey](https://chocolatey.org)`

File: README.md [hunk 2]
```diff
@@ -15,6 +15,7 @@
   - [Windows](#windows)
     - [Portable zip tarball (recommended)](#portable-zip-tarball-recommended)
     - [Installer](#installer)
+    - [Using Chocolatey?](#chocolatey)
```
Comment (PeterDaveHello) [inline README.md]: And this would be `[Using Chocolatey](#using-chocolatey)`

Comment (mom1) [reply README.md]: This not work

Comment (mom1) [reply README.md]: because link in header

Comment (mom1) [reply README.md]: Work! Magic. :smile:

**Comment by PeterDaveHello** [comment]
Oops, that's an accident.

**Comment by PeterDaveHello** [comment]
@mom1 seems there is an issue so that I can't reopen this PR, may I know if you're using GitHub WebGUI or some other git clients locally to commit the changes? Thanks.

**Comment by mom1** [comment]
I use the console client to commit changes. And WebGUI for PR/

**Comment by PeterDaveHello** [comment]
Thanks, would you please try `git push origin +7d3a7a1:master` to update this branch, and try to reopen it in WebGUI?

**Comment by mom1** [comment]
I can not reopen #1113

**Comment by PeterDaveHello** [comment]
Thank you @mom1

**Comment by mom1** [comment]
:v:

**Pull request closed without merging**